defs.h

```diff
@@ -38,6 +38,9 @@
 // Item indestructible durability
 #define DUR_INDESTRUCTIBLE		255
 
+#define VOLUME_MIN				-1600;

```

Comment:
No semicolon here

Broke build: https://ci.appveyor.com/project/galaxyhaxz/devilution/build/1.0.700-yrvlsiax

Reply:
OMG I am going to go to bed now, brain not working one bit today.

P.s. I fixed this first...

Reply:
> OMG I am going to go to bed now, brain not working one bit today.

Hehe, sleep tight. I'll merge once the build is fixed :)